functest/send_money_test.go

```diff
@@ -143,11 +147,10 @@ func TestTransferToMyself(t *testing.T) {
 	member := createMember(t, "Member1")
 	oldMemberBalance := getBalanceNoErr(t, member, member.ref)
 
-	amount := 100
-
-	_, err := signedRequest(member, "Transfer", amount, member.ref)
-	require.Contains(t, err.Error(), "[ transferCall ] Recipient must be different from the sender")
+	amount := "20"
 
+	_, err := signedRequest(member, "wallet.transfer", map[string]interface{}{"amount": amount, "to": member.ref})
+	require.NotNil(t, err)
```

Comment:
Its better to check error message here also, to be sure its about "recipient must be different from the sender"", not smth else